toponetx/datasets/mesh.py

```diff
@@ -126,3 +141,66 @@ def shrec_16(size: Literal["full", "small"] = "full"):
     raise ValueError(
         f"Files couldn't be found in folder {DIR}, fail to load the dataset."
     )
+
+
+def coseg(data: Literal["alien", "vase", "chair"] = "alien"):
+    """Load coseg mesh segmentation datasets".
+
+    Parameters
+    ----------
+    size : {"alien", "vase","chair"}, default='alien'
+        The name of the coseg dataset to be loaded. Options are "alien", "vase", or"chair".
+
+    Returns
+    -------
+    npz file
+        The npz files store the complexes of coseg segmentation dataset along
+        with their nodes, edges and faces features.
+
+    Notes
+    -----
+    Each npz file stores 5 keys:
+    "complexes",label","node_feat","edge_feat" and "face_feat".
+    complex : stores the simplicial complex of the mesh
+    node_feat : stores 6 dim node feature vector: position and normal of the each node in the mesh
+    edge_feat : stores 10 dim edge feature vector: diheral angle, edge span, 2 edge angle in the triangle, 6 edge ratios.
+    face_feat : face area, face normal, face angle
+    face label :  stores the label of mesh segmentation as a face label
+
+    Data Source
+    -----------
+    The coseg dataset was downloaded and processed from the repo : https://github.com/Ideas-Laboratory/shape-coseg-dataset
+
+    Example
```

Comment:
The headline is always `Examples` as per NumPy's documentation style, even if there is only one example. This will also render properly on the website.